AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -224,6 +226,14 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   // and should not trigger a relayout.
   _ignoreMaxSizeChange = CGSizeEqualToSize(_maxSizeForNodesConstrainedSize, CGSizeZero);
   
+  // Register the default layout inspector delegate for flow layouts only, custom layouts
+  // will need to roll their own ASCollectionViewLayoutInspecting implementation and set a layout delegate
+  if ([layout asdk_isFlowLayout]) {
+    _layoutDelegate = [self flowLayoutInspector];
```

Comment:
Can you evaluate the names _layoutDelegate vs _flowLayoutInspector?  Especially since they are the same object in the flow layout case, and because the _layoutDelegate is referred to as a layout inspector here, can we name them congruently?

How about _layoutInspector or _flowLayoutDelegate?